rasa/core/training/interactive.py

```diff
@@ -797,7 +797,7 @@ def _collect_actions(events: List[Dict[Text, Any]]) -> List[Dict[Text, Any]]:
 
     # need to guess the format of the file before opening it to avoid a read
     # in a write
-    if _guess_format(export_nlu_path) in {"md", "unk"}:
+    if guess_format(export_nlu_path) in {"md", "unk"}:
```

Comment:
I know this wasn't done by you, but in general we try to not import functions by their name but rather by their module so that it would e.g. be `loading.guess_format`

Reply:
Changed to `loading.guess_format`